tests/Testing/Console/RouteListCommandTest.php

```diff
@@ -70,15 +70,25 @@ public function testDisplayRoutesForCliInVerboseMode()
                 //
             })->name('user.show')->middleware('web');
         });
+        $this->router->view('view', 'welcome');
+        $this->router->redirect('about', 'about-us');
 
         $this->artisan(RouteListCommand::class, ['-v' => true])
             ->assertSuccessful()
             ->expectsOutput('')
+            ->expectsOutput('  ANY        about ........... Illuminate\\Routing\\RedirectController')
+            ->expectsOutput('             └ about-us 302')
             ->expectsOutput('  GET|HEAD   closure ............................................... ')
+            ->expectsOutput('             └ ' . __FILE__ . ':62')
             ->expectsOutput('  POST       controller-invokable Illuminate\\Tests\\Testing\\FooController')
+            ->expectsOutput('             └ ' . __FILE__ . ':112')
             ->expectsOutput('  GET|HEAD   controller-method/{user} Illuminate\\Tests\\Testing\\FooController@show')
+            ->expectsOutput('             └ ' . __FILE__ . ':107')
             ->expectsOutput('  GET|HEAD   {account}.example.com/user/{id} ............. user.show')
             ->expectsOutput('             ⇂ web')
+            ->expectsOutput('             └ ' . __FILE__ . ':69')
```

Comment:
These tests rely on line numbers within the test - so can jump around... Happy for suggestions here...

Reply:
Have switched to fixtures